docs/admin/deployment.md

```diff
@@ -1552,3 +1552,17 @@ the `[Service]` section before reloading systemd and restarting the
 [^5]: https://github.com/pmem/ndctl/issues/130
 
 [6]: <../dev/development.md#building-optional-components> (Building DAOS for Development)
+
+## Multi-user DFuse setup
+
+Running a single-user dfuse instance, for example on a compute node, requires no special setup
+however configuration is required for allowing multi-user dfuse on a node.
```

Comment:
You're probably tired of my comments on this, and to be fair it's such a common "mistake" that grammatically it's probably okay, but technically
```suggestion
Running a single-user dfuse instance, for example on a compute node, requires no special setup.
However, configuration is required for allowing multi-user dfuse on a node.
```
or
```suggestion
Running a single-user dfuse instance, for example on a compute node, requires no special setup;
however, configuration is required for allowing multi-user dfuse on a node.
```